src/services/refactors/installTypesForPackage.ts

```diff
@@ -12,7 +12,8 @@ namespace ts.refactor.installTypesForPackage {
     registerRefactor(installTypesForPackage);
 
     function getAvailableActions(context: RefactorContext): ApplicableRefactorInfo[] | undefined {
-        if (context.program.getCompilerOptions().noImplicitAny) {
+        const options = context.program.getCompilerOptions();
+        if (options.noImplicitAny || options.strict) {

```

Comment:
What about `strict: true` with `noImplicitAny: false`?

Reply:
fix up in https://github.com/Microsoft/TypeScript/pull/19427